helm-chart/test.values.yaml

```diff
@@ -12,26 +12,62 @@ cvat:
         - name: DJANGO_SETTINGS_MODULE
           value: cvat.settings.testing_rest
     worker:
-      import:
-        replicas: 1
-        additionalVolumeMounts:
-        - mountPath: /home/django/share
-          name: cvat-backend-data
-          subPath: share
-      export:
-        replicas: 1
+      additionalEnv:
+      - name: DJANGO_SETTINGS_MODULE
+        value: cvat.settings.testing_rest
+      - name: CLICKHOUSE_HOST
+        valueFrom:
+          secretKeyRef:
+            name: cvat-analytics-secret
+            key: CLICKHOUSE_HOST
+      - name: CLICKHOUSE_PORT
+        valueFrom:
+          secretKeyRef:
+            name: cvat-analytics-secret
+            key: CLICKHOUSE_PORT
+      - name: CLICKHOUSE_USER
+        valueFrom:
+          secretKeyRef:
+            name: cvat-analytics-secret
+            key: CLICKHOUSE_USER
+      - name: CLICKHOUSE_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: cvat-analytics-secret
+            key: CLICKHOUSE_PASSWORD
```

Comment:
Why set this only for the test? I think we should just add the Clickhouse variables to `cvat.sharedBackendEnv`.